framework/include/convergence/PostprocessorConvergence.h

```diff
@@ -29,4 +29,16 @@ class PostprocessorConvergence : public IterationCountConvergence
 
   /// Tolerance to which post-processor is compared
   const Real _tol;
+
+  /// Maximum number of diverging iterations
+  const unsigned int _max_diverging_iterations;
+
+  /// Diverging relative reduction
+  const Real _div_rel_reduction;
+
+  /// Current diverging iterations count
+  unsigned int _diverging_iterations;
+
+  /// Old post-processor value
```

Comment:
let s define Old here, it s not the old from getPPValueOld, it s the old from hte last time PPCV was executed